### PR TITLE
Add D3V3XA4B10LP Symbol

### PR DESCRIPTION
**Pull request by eclecticc**
Adding a symbol for the Diodes Inc D3V3XA4B10LP 4-Channel TVS Array.
https://www.diodes.com/assets/Datasheets/D3V3XA4B10LP.pdf

Copied from the pin-compatible TPD4EUSB30.

image

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by chschlue** [comment]
TI's DQA and Diodes' UDFN are not quite the same package.
Please add a matching footprint.

**Comment by eclecticc** [comment]
Added new footprint in https://github.com/KiCad/kicad-footprints/pull/1978

**Comment by cpresser** [comment]
pin length does seem wrong when looking at the screenshot.

**Comment by eclecticc** [comment]
@cpresser I kept the symbol the same as the existing TPD4EUSB30 one since they are functionally compatible for many applications.  I can shorten the pins to match the current KLC if that is preferred.

File: Power_Protection.dcm [hunk 1]
```diff
@@ -12,6 +12,12 @@ K ESD Protection diodes transient suppressor
 F http://www.onsemi.com/pub_link/Collateral/CM1213A-D.PDF
 $ENDCMP
 #
+$CMP D3V3XA4B10LP
+D 4-Channel Low Capacitance TVS Diode Array
```
Comment (chschlue) [inline Power_Protection.dcm]: Add package to description

File: Power_Protection.lib [hunk 2]
```diff
@@ -110,6 +110,70 @@ X VN 3 200 0 100 L 50 50 1 1 P
...
 #
+# D3V3XA4B10LP
+#
+DEF D3V3XA4B10LP U 0 0 Y Y 1 F N
+F0 "U" -25 350 50 H V C CNN
+F1 "D3V3XA4B10LP" 350 -400 50 H V C CNN
+F2 "Package_DFN_QFN:Diodes_UDFN-10_1.0x2.5mm_P0.5mm" -950 -400 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ UDFN*2.5x1.0mm*P0.5mm*
```
Comment (chschlue) [inline Power_Protection.lib]: Prepend Diodes*

**Comment by eclecticc** [comment]
Thanks, corrected.

File: Power_Protection.lib [hunk 3]
```diff
@@ -110,6 +110,70 @@ X VN 3 200 0 100 L 50 50 1 1 P
...
 #
+# D3V3XA4B10LP
+#
+DEF D3V3XA4B10LP U 0 0 Y Y 1 F N
+F0 "U" -25 350 50 H V C CNN
+F1 "D3V3XA4B10LP" 350 -400 50 H V C CNN
+F2 "Package_DFN_QFN:Diodes_UDFN-10_1.0x2.5mm_P0.5mm" -950 -400 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ Diodes*UDFN*2.5x1.0mm*P0.5mm*
```
Comment (chschlue) [inline Power_Protection.lib]: Fix dimensions in FP filter (WxH)

**Comment by eclecticc** [comment]
Thanks, corrected.

**Comment by chschlue** [comment]
Thanks

**Pull request merged**